### PR TITLE
Addition of developmental references

### PR DESCRIPTION
**Pull request by A1Aadi**
Added new reference links to frontend, backend, widgets and IVT in README.md. 
Added unordered list for links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,8 @@ We created this project for our own internal experiments. Once completed and sta
 
 # Development
 
-You can find further on each section below
+You can find further on each section below
```
Comment (m5nv) [inline README.md:63]: Please change this to read:
Follow the links below to get started.

**Review by m5nv — CHANGES_REQUESTED** [review]
Hi @A1Aadi,

Congrats on your first PR, and thank you! 

A few nitpicks. Change the intro text to read as commented and make links to be a proper list. You can take a look at the `Features` section on how bullets are done using `Markdown`.

Cheers!

**Comment by A1Aadi** [comment]
Finished adding list for the links

**Comment by m5nv** [comment]
Thanks!

**Pull request merged**